src/SFA.DAS.AANHub.Application/Common/Validators/RequestedByMemberId/RequestedByMemberIdValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using static SFA.DAS.AANHub.Domain.Common.Constants;
+
+namespace SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId
+{
+    public class RequestedByMemberIdValidator : AbstractValidator<IRequestedByMemberId>
+    {
+        public const string RequestedByMemberIdEmptyErrorMessage = "RequestedByMemberId can't be empty";
```

Comment:
opetional suggestion: change message to 'RequestedByMemberId is empty'

Reply:
Changed as suggested [here](https://github.com/SkillsFundingAgency/das-aan-hub-api/commit/95541f6b42ff7599e4def523e8f11135d155d44b#diff-f3479137093a22cc0db73b65d7d1a482b4ea87e3f625af48163b391058845e0d)